src/main/java/org/opensearch/sdk/TransportActions.java

```diff
@@ -26,7 +26,7 @@
  */
 public class TransportActions {
     private final Logger logger = LogManager.getLogger(TransportActions.class);
-    private Map<String, Class> transportActions;
+    private Map<String, Class<?>> transportActions;
```

Comment:
While `<?>` makes the warning go away, it still means "any class".  

However, we have more information about the class type that will be assigned.   See the constructor below:
```
Map<String, Class<? extends TransportAction<Request, Response>>> transportActions
```
So that is the generic we should use.

Also note in the constructor those `Request` and `Response` are further bounded:
```
<Request extends ActionRequest, Response extends ActionResponse>
```
So this definition here should be:
```java
Class<? extends TransportAction<? extends ActionRequest, ? extends ActionResponse>>
```

Reply:
Thanks @dbwiddis